internal/commands/network/create.go

```diff
@@ -43,11 +49,8 @@ func (s *createCommand) InitCommand() {
 		"  gateway: string \n"+
 		"  dhcp: true/false \n"+
 		"  dhcp-default-route: true/false \n"+
-		"  dhcp-dns: array of strings \n"+
-		"Usage: \n"+
-		"	--ip-network 'address=94.23.112.143,\"dhcp-dns=<value1>,<value2>\",gateway=<gateway>,dhcp=true' \n"+
-		"	--ip-network address=94.43.112.143/32,dhcp-dns=<value>\n")
-	s.AddFlags(fs) // TODO(ana): replace usage with examples once the refactor is done.
+		"  dhcp-dns: array of strings \n")
```

Comment:
is the tailing newline necessary/standard? This could also be backquoted, I guess, although that would lead to kind of an ugly look..

Reply:
Honestly not sure, we could probably delete those new lines as they don't work anyway (everything prints in one line in my terminal)... But that sounds like a separate MR, since there's a number of those that would need fixing